include/astshim/ChebyMap.h

```diff
@@ -0,0 +1,321 @@
+    polynomial, the best fitting maximum-order polynomial is returned so
+    long as its accuracy is better than
+    "maxacc". If it is not, an error is reported.
+
+    @param[in] forward  If true the forward transformation is replaced.
+                    Otherwise the inverse transformation is replaced.
+    @param[in] acc  The target accuracy, expressed as a geodesic distance within
+                    the ChebyMap's input space (if `forward` is false)
+                    or output space (if `forward` is true).
+    @param[in] maxacc  The maximum allowed accuracy for an acceptable polynomial,
+                    expressed as a geodesic distance within the ChebyMap's input space
+                    (if `forward` is false) or output space (if `forward` is true).
+    @param[in] maxorder  The maximum allowed polynomial order. This is one more than the
+                    maximum power of either input axis. So for instance, a value of
+                    3 refers to a quadratic polynomial.
+                    Note, cross terms with total powers greater than or equal to `maxorder`
+                    are not inlcuded in the fit. So the maximum number of terms in
+                    each of the fitted polynomials is `maxorder*(maxorder + 1)/2.`
+    @param[in] lbnd  A vector holding the lower bounds of a rectangular region within
+                    the ChebyMap's input space (if `forward` is false)
+                    or output space (if `forward` is true).
+                    If both lbnd and ubnd are empty (the default) then they will be estimated.
+                    The new polynomial will be evaluated over this rectangle. The length
+                    should equal getNin() or getNout(), depending on `forward`.
+    @param[in] ubnd  A vector holding the upper bounds of a rectangular region within
+                    the ChebyMap's input space (if `forward` is false)
+                    or output space (if `forward` is true).
+                    If both lbnd and ubnd are empty (the default) then they will be estimated.
+                    The new polynomial will be evaluated over this rectangle. The length
+                    should equal getNin() or getNout(), depending on `forward`.
+
+    @throw std::invalid_argument if the size of `lbnd` or `ubnd` does not match getNin() (if `forward` false)
+                    or getNout() (if `forward` true).    
+
+    The variant that takes omits the `lbnd` and `ubnd` arguments
+    uses the full domain of the polynomial whose inverse is being fit.
+    @{
+    */
+    ChebyMap polyTran(bool forward, double acc, double maxacc, int maxorder, std::vector<double> const &lbnd,
+                      std::vector<double> const &ubnd) const;
+    ChebyMap polyTran(bool forward, double acc, double maxacc, int maxorder) const;
+    ///@}
+
+protected:
+    virtual std::shared_ptr<Object> _copyPolymorphic() const override {
```

Comment:
Are you making a suggestion? Note that all Mappings are required to override this, and all do so in the same way.